src/Blaster.WebApi/Features/Shared/_Layout.cshtml

```diff
@@ -43,22 +42,27 @@
                     <span aria-hidden="true"></span>
                     <span aria-hidden="true"></span>
                     <span aria-hidden="true"></span>
-                </a>        
+                </a>
             </div>
         </div>
-        
+
         <div id="menu" class="navbar-menu">
             <div class="navbar-start">
                 <a href="@Url.Action("Index", "Frontpage")" class="navbar-item">Home</a>
-                <a href="@Url.Action("Index", "Capability")" class="navbar-item">Capabilities</a>             
-                <a href="https://playbooks.dfds.cloud" class="navbar-item">Playbooks</a> 
+                <a href="@Url.Action("Index", "Capability")" class="navbar-item">Capabilities</a>
+                <a href="https://playbooks.dfds.cloud" class="navbar-item">Playbooks</a>
                 <a href="https://github.com/dfds" class="navbar-item">GitHub (Code examples)</a>
                 <a href="@Url.Action("SharedComponents", "Frontpage")" class="navbar-item">UI</a>
             </div>
 
             <div class="navbar-end">
-                <div class="navbar-item">
-                    @UserHelper.CurrentUser.Name
+                <div v-cloak class="navbar-item">
+                    <a v-if="isAuthenticated()" v-on:click="signOut()">
+                        {{getUserName() }}
+                    </a>
+                    <a v-if="!isAuthenticated()" v-on:click="signIn()">
+                        {{getUserName() }}

```

Comment:
While the navbar seems to be aware of when a user has been logged in, and displays everything related to the user correctly, the rest of the page doesn't. E.g. when I navigate to '/capabilities' it says that I have joined the same Capabilities as the previously hardcoded test user (jdog@me.com). Also, if one isn't logged in, the navbar seems to default to whatever user is provided by UserHelper. Intended behaviour? If it isn't, we may want to display some text or a button here instead, indicating that the user can click on said object to log-in with.